Documentation/devel/stage1-implementors-guide.md

```diff
@@ -95,6 +95,12 @@ For example, it removes the network namespace of a pod.
 * `--debug` to activate debugging
 * UUID of the pod
 
+### `rkt stop` => "coreos.com/rkt/stage1/stop"
```

Comment:
It needs documentation in https://github.com/coreos/rkt/tree/master/Documentation/subcommands too.